crates/graphql/src/schema/objects/creator.rs

```diff
@@ -22,13 +24,58 @@ struct AttributeGroup {
     name: String,
     variants: Vec<AttributeVariant>,
 }
+#[derive(Debug, Clone)]
+struct CreatorCounts {
+    creator: Creator,
+}
+
+impl CreatorCounts {
+    #[must_use]
+    pub fn new(creator: Creator) -> Self {
+        Self { creator }
+    }
+}
+
+#[graphql_object(Context = AppContext)]
+impl CreatorCounts {
+    fn creations(&self, context: &AppContext) -> FieldResult<i32> {
+        let conn = context.db_pool.get()?;
+
+        let count = metadata_creators::table
+            .filter(metadata_creators::creator_address.eq(&self.creator.address))
+            .filter(metadata_creators::verified.eq(true))
+            .count()
+            .get_result::<i64>(&conn)?;
+
+        Ok(i64::try_into(count)?)
+    }
+}
 
 #[graphql_object(Context = AppContext)]
 impl Creator {
     fn address(&self) -> &str {
         &self.address
     }
 
+    fn counts(&self) -> CreatorCounts {
+        CreatorCounts::new(self.clone())
+    }
+
+    #[graphql(arguments(auction_houses(description = "Auction house public keys")))]
+    pub async fn stats(
+        &self,
+        auction_houses: Vec<PublicKey<AuctionHouse>>,
+        ctx: &AppContext,
+    ) -> FieldResult<Vec<MintStats>> {
+        let conn = ctx.db_pool.get()?;
+        let rows = stats::collection(&conn, auction_houses, &self.address)?;
```

Comment:
Should this be a data loader?

Reply:
Yes but we only load 1 creator at a time and its not a clean batch. need auction_houses to be fixed across the creators.